course-mw/sunbird-util/sunbird-platform-core/common-util/src/main/resources/externalresource.properties

```diff
@@ -207,3 +207,5 @@ enrollment_list_size=1000
 sunbird_course_batch_path=/app/toc/{courseId}/overview?batchId={batchId}
 enrolment_list_include_retired_courses=false
 enrol_end_date_allow_lesser_value=true
+enroll_collection_allow_primary_category=Program
+program_enroll_allow_primary_category=Course
```

Comment:
these variable names are not user friendly...

Reply:
Done